documentation/src/main/asciidoc/topics/upgrading.adoc

```diff
@@ -49,6 +49,14 @@ The same configuration should be changed to the following instead.
 </persistence>
 ----
 
+== RocksDB store
+
+The RocksDB implementation is no longer included by default with the server. It can be installed via the CLI:
```

Comment:
Should this had said RocksDB JNI implementation?

Reply:
AFAIK, there isn't another one: https://repo1.maven.org/maven2/org/rocksdb/